### ISSUE TITLE
Support ES5

### ISSUE THREAD

**Issue by jgornick** [bodia-uz/odata-filter-builder#13]
The library is distributed with ES6 targeted, but to support IE11, would it be possible to target ES5?

**Comment by bodia-uz** [comment]
Hi @jgornick.
Library distributed with es5:
1. umd:
https://unpkg.com/odata-filter-builder@1.0.0-0/dist/odata-filter-builder.js
2. commonjs: 
https://unpkg.com/odata-filter-builder@1.0.0-0/lib/ODataFilterBuilder.js

Could you describe what is wrong in IE?

**Comment by jgornick** [comment]
I suppose when I'm using TypeScript and importing like:

```typescript
import { ODataFilterBuilder } from 'odata-filter-builder';
```

My output via webpack is using the `src/ODataFilterBuilder.js` defined via `module` in `package.json`. This should be using the `main` `lib/ODataFilterBuilder.js`.

If I remove `module` then my build uses the ES5 version of the script in `lib`.

Thoughts?

**Comment by bodia-uz** [comment]
I see.
For `module` field, we need transpile our source to es5, with `import/export` declaration (https://rollupjs.org/guide/en#output-format-f-format)

As a workaround, you could include `odata-filter-builder` to your typescript loader, or remove `module` field from [`mainFields`](https://webpack.js.org/configuration/resolve/#resolve-mainfields):
```
mainFields: ["main"]
```

Fill free to create PR.
It should be something like:
```
"module": "es/ODataFilterBuilder.js",
...
"build:es": "rollup -c -f es src/ODataFilterBuilder.js > es/ODataFilterBuilder.js",
```

**Comment by jgornick** [comment]
I ended up fixing in my webpack config like:

```
resolve: {
  alias: {
    'odata-filter-builder': path.resolve(
      helpers.root('node_modules'),
      'odata-filter-builder',
      'lib',
      'ODataFilterBuilder.js'
    )
  }
}
```

However, I do agree that by default the `module` and/or `main` in the distributed package.json point to an ES5 version.

I'll try to get something together for this and submit a PR.

**Comment by bodia-uz** [comment]
Published version [1.0.0-1](https://unpkg.com/odata-filter-builder@1.0.0-1/es/ODataFilterBuilder.js) with fix.
